conans/client/rest/remote_credentials.py

```diff
@@ -66,3 +81,9 @@ def _get_env(remote, user):
         if passwd:
             ConanOutput().info("Got password '******' from environment")
         return user, passwd
+
+def _load_auth_plugin(auth_plugin_path):
+    if os.path.exists(auth_plugin_path):
+        mod, _ = load_python_file(auth_plugin_path)
+        if hasattr(mod, "auth_plugin"):
+            return mod.auth_plugin
```

Comment:
I would change this name to `auth_remote_plugin` to better differenciate it from the `auth_sources_plugin` one and make the naming more uniform

Reply:
Then, I wonder if it'd be better to name it `auth_remote_plugin`. Something like:

```python
    @property
    def auth_remote_plugin_path(self):
        return os.path.join(self._home, _EXTENSIONS_FOLDER, _PLUGINS, "auth_remote.py")

    @property
    def auth_source_plugin_path(self):
        return os.path.join(self._home, _EXTENSIONS_FOLDER, _PLUGINS, "auth_source.py")
```